libs/src/index.js

```diff
@@ -268,7 +271,9 @@ class AudiusLibs {
       this.contracts = new AudiusContracts(
         this.web3Manager,
         this.web3Config ? this.web3Config.registryAddress : null,
-        this.isServer)
+        this.isServer,
+        this.enableUserReplicaSetManagerContract
```

Comment:
where is this boolean stored in src/services/dataContracts?

Reply:
fixed